solutions/secure-cross-regional-bucket/variables.tf

```diff
@@ -59,7 +59,7 @@ variable "existing_cos_instance_id" {
 
 variable "bucket_access_tags" {
   type        = list(string)
-  description = "A list of access tags to apply to the Cloud Object Storage instance bucket."
+  description = "A list of access tags to apply to the Object Storage instance created by the module. [Learn more](https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial)."
```

Comment:
this is for bucket tags, not instance

Reply:
updated with https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-object-tagging

Also updated in `secure-regional-bucket`